packages/eos-components/src/utils.js

```diff
@@ -68,7 +68,11 @@ export function getNodeUrl(node, channelId) {
 
 export function getLeaves(node) {
   if (!node.children) {
-    return [node];
+    if (node.kind === 'topic') {
+      return [];
+    } else {
```

Comment:
This else is not needed after a return: https://eslint.org/docs/rules/no-else-return

Reply:
OK! I can change this, I prefer that too. But that eslint rule is not complaining in our checks.